pkgs/games/itch-setup/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, writeShellScriptBin, steam-run, fetchurl }:
+let
+
+  pname = "itch-setup";
+  version = "1.26.0";
+
+  src = fetchurl {
+    url = "https://broth.itch.ovh/itch-setup/linux-amd64/${version}/unpacked/default";
+    hash = "sha256-bcJKqhgZK42Irx12BIvbTDMb/DHEOEXljetlDokF7x8=";
+    executable = true;
+  };
+
+in
+(writeShellScriptBin pname ''exec ${steam-run}/bin/steam-run ${src} "$@"'') // {
+
+  passthru = { inherit pname version src; };
```

Comment:
```suggestion
```
That is done by default.

Reply:
https://github.com/NixOS/nixpkgs/blob/93e7e038c35a14f6afa36e768d9fac2afe572c45/pkgs/build-support/trivial-builders.nix#L116

The attribute is called `name`, not `pname` and neither`src` nor `version` arepassed by default.

Reply:
oh, yeah only for mkDerivation, not for writeShellScriptBin.